convert_json_to_yaml.py

```diff
@@ -0,0 +1,10 @@
+from pathlib import Path
```

Comment:
Does this file really need to be here, or was it just used to do the initial conversion? (I don't see it referenced anywhere.)

Reply:
Committed by mistake.  Will remove.